### PR TITLE
Add documentation for newly added Switch and Scene support for Lutron.

### PR DESCRIPTION
**Pull request by cdheiser**
**Description:**
Added documentation to accompany added support for switches and scenes for Lutron.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18330

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/scene.lutron.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+description: "Instructions on how to setup the Lutron scenes within Home Assistant."
+date: 2018-10-09 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lutron.png
+ha_category: Scene
+ha_iot_class: "Local Polling"
+---
```
Comment (MartinHjelmare) [inline source/_components/scene.lutron.markdown:14]: Add `ha_release` item. It should have 0.85 as value.

File: source/_components/switch.lutron.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+description: "Instructions on how to setup the Lutron switches within Home Assistant."
+date: 2018-10-09 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lutron.png
+ha_category: Switch
+ha_iot_class: "Local Polling"
+---
```
Comment (MartinHjelmare) [inline source/_components/switch.lutron.markdown:14]: Same as above.

Comment (frenck) [reply source/_components/switch.lutron.markdown:14]: Done.

**Review by frenck — APPROVED** [review]
LGTM. Added missing `ha_release`.

**Pull request merged**